src/Symfony/Component/Messenger/Bridge/AmazonSqs/Transport/AmazonSqsTransport.php

```diff
@@ -26,14 +26,14 @@
 /**
  * @author Jérémy Derussé <jeremy@derusse.com>
  */
-class AmazonSqsTransport implements TransportInterface, SetupableTransportInterface, MessageCountAwareInterface, ResetInterface
+class AmazonSqsTransport implements TransportInterface, KeepaliveReceiverInterface, SetupableTransportInterface, MessageCountAwareInterface, ResetInterface
 {
     private SerializerInterface $serializer;
 
     public function __construct(
         private Connection $connection,
         ?SerializerInterface $serializer = null,
-        private (ReceiverInterface&MessageCountAwareInterface)|null $receiver = null,
+        private (KeepaliveReceiverInterface&MessageCountAwareInterface)|null $receiver = null,
```

Comment:
this change making the constructor argument type stricter than before is a BC break.

Reply:
Right, reverted